Cargo.toml

```diff
@@ -17,7 +17,8 @@ byteorder = "1.0"
 lazy_static = "0.2.1"
 tinysegmenter = "0.1.0"
 regex = "0.2"
-fst = {version="0.2", default-features=false}
+fst = {version="0.3", default-features=false}
+fst-levenshtein = "0.2"
```

Comment:
Should this be optional - maybe behind a `fuzzy` feature gate until the whole feature is done?

Reply:
we can remove `fst-levenshtein` and keep only `levenshtein-automata`. The latter does the same thing, but faster, better and stronger.